open_spiel/python/algorithms/alpha_zero/alpha_zero.py

```diff
@@ -194,6 +201,8 @@ def _init_bot(config, game, evaluator_, evaluation):
       dirichlet_noise=noise,
       child_selection_fn=mcts.SearchNode.puct_value,
       verbose=False)
+  
```

Comment:
probably worth adding an else: raise

Reply:
Do I need a raise, since I am checking the flag on the example file?